packages/react-storage/tsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
   "extends": "@aws-amplify/typescript-config/tsconfig.react.json",
-  "include": ["*.mjs", "*.ts", "src/**/*.ts", "src/**/*.tsx"],
+  "include": ["**/*.mjs", "**/*.ts", "src/**/*.ts", "src/**/*.tsx"],
```

Comment:
This change was required to get TS + ESlint to be able to see the exports test file

Reply:
Think we can avoid this if we put the tests under _src_?

Reply:
good idea, will move them